Sources/SwiftProtobuf/AsyncMessageSequence.swift

```diff
@@ -13,7 +13,8 @@
 ///
 // -----------------------------------------------------------------------------
 
-@available(macOS 12.0, iOS 15.0, tvOS 15.0, watchOS 8.0, *)
+#if swift(>=5.5) && canImport(_Concurrency)
+@available(macOS 10.15, *)
```

Comment:
This needs the full range of availability annotations i.e. iOS, tvOS and watchOS as well

Reply:
Yes, I'll use the same annotation as on `AsyncSequence`. Can we get rid of the Swift version cards @thomasvl given what you just committed?

Reply:
> Yes, I'll use the same annotation as on `AsyncSequence`. Can we get rid of the Swift version cards @thomasvl given what you just committed? 

Yup, I'm working on the changes now to drop the gates in the rest of the library, so you can also drop the sendable wrapper also and directly use `Sendable`.  (I was going to keep the wrapper in my PR so we weren't racing and clean it up afterwards if need be)